priv/migrations/20241011120000_restructure_aexn_invalid_contracts.ex

```diff
@@ -0,0 +1,40 @@
+defmodule AeMdw.Migrations.RestructureAexnInvalidContracts do
+  @moduledoc """
+  Generate block difficulty statistics.
+  """
+  alias AeMdw.Collection
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.State
+  alias AeMdw.Db.WriteMutation
+
+  require Model
+
+  import Record
+
+  defrecord :aexn_invalid_contract, [:index, :reason]
+
+  @spec run(State.t(), boolean()) :: {:ok, non_neg_integer()}
+  def run(state, _from_start?) do
+    state
+    |> Collection.stream(Model.AexnInvalidContract, nil)
+    |> Stream.map(&State.fetch!(state, Model.AexnInvalidContract, &1))
+    |> Stream.filter(&match?(aexn_invalid_contract(), &1))
+    |> Stream.map(fn aexn_invalid_contract(
+                       index: index,
+                       reason: reason
+                     ) ->
+      WriteMutation.new(
+        Model.AexnInvalidContract,
+        Model.aexn_invalid_contract(index: index, reason: reason, description: "unknown")
```

Comment:
We have one more place where we mark a contract as invalid ( https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/db/sync/stats.ex#L56 ), which the current PR leaves with default description (i.e. `nil`). We should probably have all the contracts without specific description as `unknown` (add description to the other place where contracts are being marked as invalid) or nil (remove the migration).